Plugins/Flow.Launcher.Plugin.Shell/Settings.cs

```diff
@@ -36,6 +36,7 @@ public enum Shell
         Cmd = 0,
         Powershell = 1,
         RunCommand = 2,
+        Terminal = 3,
```

Comment:
1. Would it make sense for this to be named `Windows Terminal` ?
2. Do we need translations for this?

Reply:
It's just called Terminal, and flow launcher is windows only, so I don't think the distinction is necessary. As for translations, the other options aren't translated, so it kind of be adding too much responsibility to this PR.